yarn.lock

```diff
@@ -109,7 +109,7 @@
 
 "@babel/runtime@7.18.9":
   version "7.18.9"
-  resolved "https://registry.npmjs.org/@babel/runtime/-/runtime-7.18.9.tgz"
```

Comment:
These other dependencies changed resolved path, since I removed local verdaccio suomifi-ui-composonents installation (and it's dependencies), and installed suomifi-ui-components from npm